src/Audit/AuditOSSIndex.spec.ts

```diff
@@ -75,4 +94,36 @@ describe('AuditOSSIndex', () => {
 
     expect(result).to.equal(false);
   });
+
+  it('should include supplemental when empty', () => {
```

Comment:
Both of these tests fail for me locally, currently.

Reply:
Could you share the failure info?